submissions/templates/submissions/detail.html

```diff
@@ -183,6 +199,59 @@ <h4 id="page-{{forloop.counter}}"></h4>
                             <textarea name="new_comment" class="form-control" id="newComment" rows="7"></textarea>
                         </div>
                     </div> {% endcomment %}
+
+                    <!--Populate Version Comments-->
+                    {% if submission.assignment.versioned == True %}
```

Comment:
As long as there is no added comments for this version, there is no need to should the label. Currently even if the grader hasn't added solution comments, the label "Version Comments" is still visible. And these take quite a lot of vertical space in the sidebar, but they don't provide that much information. I would prefer it to be a bootstrap accordion https://getbootstrap.com/docs/5.1/components/accordion/, will all items collapsed by default.